workspaces/ui-v2/package.json

```diff
@@ -15,65 +15,35 @@
     "start": "node scripts/start.js",
     "build:local": "env OPTIC_WEBPACK_ENTRYPOINT=local node scripts/build.js",
     "build:cloud": "env OPTIC_WEBPACK_ENTRYPOINT=cloud node scripts/build.js",
-    "build": "build:local",
+    "build": "yarn run build:local",
```

Comment:
I don't believe this command is directly called, but it wasn't working for me so i updated to what i think it should call